go.mod

```diff
@@ -62,6 +62,7 @@ replace (
 	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.1.0
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 	github.com/hashicorp/go-retryablehttp => github.com/hashicorp/go-retryablehttp v0.7.7
+	github.com/microsoft/moc => github.com/microsoft/moc v0.25.2-0.20241212023151-07f45e6a42e2
```

Comment:
Is this going to be changed to an official version before merging?

Reply:
Yes, it must be updated later once the moc repo is approved and make official build.